src/main/java/com/robinpowered/sdk/model/Organization.java

```diff
@@ -43,4 +57,23 @@ public Reference(int organizationId) {
             super(organizationId);
         }
     }
+
+    public static final class UserAccess {
+
+        private Integer managementLevel;
+        private Boolean isOwner;
+
+        public UserAccess(Integer managementLevel, Boolean isOwner) {
+            this.managementLevel = managementLevel;
+            this.isOwner = isOwner;
+        }
+
+        public int getManagementLevel() {
+            return managementLevel;
+        }
+
+        public Boolean getOwner() {
+            return isOwner;
+        }
+    }
```

Comment:
Is there a reason you want to create a custom entity for this vs. keep parity with the API's model? I think the SDK largely keeps parity.